src/amici_interface_matlabo2.cpp

```diff
@@ -0,0 +1 @@
+#include "amici_interface_matlab.cpp"
```

Comment:
Hm.. but why is it even compiled twice? The object file should be usable for both versions.